### ISSUE TITLE
Data

### ISSUE THREAD

**Issue by acerhp** [mk-minchul/AdaFace#65]
Where is the validation dataset? The path I see in the code is

**Comment by acerhp** [comment]
Can you provide a directory navigation to validate the dataset?

**Comment by acerhp** [comment]
Can you provide the directory structure about validation dataset

**Comment by zivzone** [comment]
@acerhp 
I am also curious about the protocol for validation, and I feel a little bit weird, too.
It seems that the training pipeline will pick up the model to test based on the best validation score(it seems 5 Val set avg if I don't misunderstand anything)
But I thought that the validation set and the final test set are the same based on the default configuration file, they both are the LFW and etc. Does it OK to do this? Or @mk-minchul , your guys always pick the final epoch model to test. In this way, I think everything is legal.
